py-rse/provenance.Rmd

```diff
@@ -439,11 +439,23 @@ then review the spreadsheet file and the coded response file.
 4.  This dataset is clearly in support of an article.
     What information can you find about it, and can you find a link to it?
 
+### Evaluate a project's code provenance {#provenance-ex-eval-code}
+
+The GitHub repository https://github.com/borstlab/reversephi_paper/ provides the code and data for the paper @leonhardt2017.  Browse the repository and answer the following questions:
+
+1. Where is the software environment described? What files would you need to recreate the software environment? 
+2. Where are the data processing steps described? How could you re-create the results included in the manuscript?
+3. How are the scripts and data archived?  I.e. Where can you download the version of the code and data as it was when the manuscript was published?
+
 ### Making permanent links {#provenance-ex-permanent-links}
 
 The link to the UK Home Office's [accessibility guideline posters][ukho-accessibility] might change in future.
 Use the [Wayback Machine][wayback-machine] to find a link that is more likely to be usable in the long run.
-The fourth file is normally only found in research projects:
+
+### Create a citable archive of your Zipf's analysis {#provenance-ex-zenodo}
```

Comment:
I like the idea of this exercise, my only hesitation is that we'll have a whole bunch of people creating a DOI for something that nobody will ever really cite. I guess that doesn't matter?

Reply:
Yeah, I didn't really think about that...and now it occurs to me that I should probably check Zenodo's terms of service.  Perhaps we could replace this exercise with just creating a GitHub release?

Reply:
A GitHub release instead is a good idea.